src/app/(auth)/signup/page.js

```diff
@@ -1,14 +1,34 @@
 "use client";
 
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/navigation";
 import SignUpAccount from "./components/SignUpAccount";
 import SignUpDetails from "./components/SignUpDetails";
 import SignUpEmailConfirmation from "./components/SignUpEmailConfirmation";
 import SignUpChildren from "./components/SignUpChildren";
+import { AUTH_TOKEN_NAME } from "@/app/api.mjs";
+
+const signupCurrentStepKey = 'signup_current_step';
```

Comment:
SIGNUP_CURRENT_STEP_KEY, that is syntax for constants in JS.

Reply:
your previous comment: "Shoudl be camelCase"